README.md

```diff
@@ -59,6 +59,26 @@ with:
   projectBaseDir: my-custom-directory
 ```
 
+In case you need to add additional options to the scanner, you can use the `args` option:
+
+```yaml
+- name: Analyze with SonarCloud
+  uses: sonarsource/sonarcloud-github-action@master
+  with:
+    projectBaseDir: my-custom-directory
+    args: >
+      -Dsonar.organization=my-organization
+      -Dsonar.projectKey=my-projectkey
+      -Dsonar.python.coverage.reportPaths=coverage.xml
+      -Dsonar.sources=lib/
+      -Dsonar.test.exclusions=tests/**
+      -Dsonar.tests=tests/
+      -Dsonar.verbose=true
+```
+
+More information about possible sonar-scanner parameters is found in the documentation at:
+https://docs.sonarqube.org/latest/analysis/analysis-parameters/
```

Comment:
Should be replaced by https://sonarcloud.io/documentation/analysis/analysis-parameters/

Reply:
Fixed.